lib/linter-views.coffee

```diff
@@ -131,75 +132,47 @@ class LinterViews
       )
       throw null
 
-  updateCurrentLine: (line) ->
-    if @currentLine isnt line
-      @currentLine = line
-      @updateLineMessages()
-      @renderPanel()
-
-
-  updateLineMessages: ->
-    activeEditor = atom.workspace.getActiveTextEditor()
-    @linter.eachEditorLinter (editorLinter) =>
-      return unless editorLinter.editor is activeEditor
-
-      @lineMessages = []
+  updateLineMessages: (shouldRender = false) ->
+    return unless @tabs['Line'].visibility
+    @lineMessages.clear()
+    activeTextEditor = atom.workspace.getActiveTextEditor()
```

Comment:
`activeTextEditor` looks redundant to me. Seems like if `currentLine` is undefined, then whole following block boils down to nothing anyway.
Should this be condensed to `atom.workspace.getActiveTextEditor()?.getCursorBufferPosition()?.row`?

BTW, what about case of multiple cursors? Should it be `getCursorBufferPositions` and a `forEach` over those instead? Not necessarily, but something to consider.

And... it was merged. Oh well. I'll keep looking though.